canvg.js

```diff
@@ -231,8 +231,14 @@
 			}
 			else if (window.DOMParser)
 			{
-				var parser = new DOMParser();
-				return parser.parseFromString(xml, 'text/xml');
+				try {
+					var parser = new DOMParser();

```

Comment:
the line `var parser = new DOMParser();` is duplicated, can just move it outside the `try/catch` unless somehow `parseFromString` maintains state